tests/test_dns_server_nonbreaking_changes.py

```diff
@@ -11,7 +11,7 @@
 from adafruit_wiznet5k.adafruit_wiznet5k_socket import socket
 
 
-DEFAULT_DEBUG_ON = True
+DEFAULT_DEBUG_ON = False
```

Comment:
Was this changed intentionally? I think I lean toward having debug True in the test, though it's not a strong preference. Can def keep it False if there is a reason for it.

Reply:
It was a convenience for testing, that I will refactor to remove.